### DISCUSSION TITLE
Actually I want to apply for student developer pack

### DISCUSSION THREAD

**Discussion by panu1234eng** [community/community#147597] [GitHub Education]
### Hi everyone,

Actually I want to apply for GitHub student developer pack

**Comment by davevad93** [comment]
Hi @panu1234eng ,

To qualify for student benefits, you must:

- Have a GitHub account.
- Be at least 13 years old.
- Be currently enrolled in a degree or diploma granting course of study from a recognized educational institution.
- Have a verifiable school-issued email address.
- Be able to provide documentation from your school which demonstrates your current student status.

Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.

If you meet the above requirements and would like to enroll, you can submit your application for the `Student Developer Pack` [here](https://education.github.com/discount_requests/application).
